autoware_launch/launch/e2e_simulator.launch.xml

```diff
@@ -6,6 +6,7 @@
   <arg name="vehicle_model" default="sample_vehicle" description="vehicle model name"/>
   <arg name="sensor_model" default="sample_sensor_kit" description="sensor model name"/>
   <arg name="data_path" default="$(env HOME)/autoware_data" description="packages data and artifacts directory path"/>
+  <arg name="carla" default="false" description="also launch the interface with the CARLA simulator"/>
```

Comment:
Let's change the argument to be `simulator_type`.
```suggestion
  <arg name="simulator_type" default="awsim" description="select the simulator type ('awsim' or 'carla')"/>
```

Reply:
Regarding this, XML Launch file cannot accept string as arguments of `if` on line 42. I will try to find the work around to make it work.

Reply:
Fixed, changed the arguments and the conditional launch to `eval`.